windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -366,6 +390,19 @@ function TMergeKeyboardInfo.Failed(message: string): Boolean;
   Result := False;
 end;
 
+procedure TMergeKeyboardInfo.AddJSFile(const Filename: string);
```

Comment:
If there was a compelling reason to move this, fine. If not, keeping it the same (relative) place in the file would make it easier to see what actually changed.

Reply:
I moved it because it looked like all the function bodies were before procedures